genericjpa/db/src/main/java/org/hibernate/search/annotations/UpdateInfo.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.annotations;
+
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * @author Martin Braun
+ * @hsearch.experimental
+ */
+@Target({FIELD, METHOD, TYPE})
+@Retention(RUNTIME)
+@Repeatable(UpdateInfos.class)
+public @interface UpdateInfo {
```

Comment:
To make it clear:

The plan is to keep the @UpdateInfo annotations and build a separate system that is used in conjunction with Hibernate ORM.